Testing/WinMLRunnerTest/WinMLRunnerTest.cpp

```diff
@@ -887,7 +865,7 @@ namespace WinMLRunnerTest
         TEST_METHOD(LoadModelFailModelNotFound)
         {
             const std::wstring command = BuildCommand({ EXE_PATH, L"-model", L"invalid_model_name" });
-            Assert::AreEqual(HRESULT_FROM_WIN32(ERROR_FILE_NOT_FOUND), RunProc((wchar_t *)command.c_str()));
+            Assert::AreEqual(HRESULT_FROM_WIN32(E_FAIL), RunProc((wchar_t *)command.c_str()));
         }
```

Comment:
this seems like a bug in winml - file missing should be file not found, not e_fail. can we log the bug in winml too?